src/bin/composer-requirements.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import json
+from os import path
+import sys
+
+
+COMPOSER_LOCAL = 'composer-local.json'
+
+
+def merge_requirements(env_data, local_data):
+    env_require = env_data['require']
+
+    for package in env_require:
+        if package in local_data['require'].keys():
+            local_data['require'][package]=env_require[package]
+            print('Found {0}: Replacing with {1}'.format(package, env_require[package]))
+
+    return local_data
+
+if __name__== "__main__":
+    if len(sys.argv) < 3:
+        print('Argument are missing.\n Syntax: {0} <directory> <environment>'.format(sys.argv[0]))
+        exit(1)
+
+    directory = sys.argv[1]
+    environment = sys.argv[2]
+
+    try:
+        env_file = open('{0}composer-{1}.json'.format(directory, environment), 'r')
+    except FileNotFoundError:
+        print('No environment specific requirements')
+        exit(0)
+
+    try:
+        local_file = open('{0}{1}'.format(directory, COMPOSER_LOCAL), 'r')
+    except FileNotFoundError:
+        print('No local specific requirements')
+        exit(0)
+
+    env_data = json.loads(env_file.read())
+    env_file.close()
+    local_data = json.loads(local_file.read())
+    local_file.close()
+
```

Comment:
Maybe we can not do the merge if content is identical (I just tested with "local" environment, it report some merge even if it's the same file, which looks a bit confusing), something like 
```python


    if env_data == local_data:
        print('No difference between files.')
        exit(0)
```

Reply:
When doing the tests locally, I had a simplified version for environment file that only holds the `require` array. See [cidev example](https://github.com/greenpeace/planet4-cidev/blob/main/composer-production.json). Maybe we can drop the `composer-` prefix to avoid confusion? 🤔

Reply:
Haaa good point :+1:  so my remark was more about the `Found {0}: Replacing with {1}` message showing even if the value is the same for the local and env require line (but it's mostly decorative, shouldn't matter too much)